cocoapods-patch.gemspec

```
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.files         = Dir['*.md', 'lib/**/*', 'LICENSE']
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency "cocoapods", "~> 1.12.0"
+  spec.add_runtime_dependency "cocoapods", "~> 1.13.0"
```

Wondering if in light of 1.14.2 being released and this very much pins this library to 1.13.x, we could instead do this to prevent these iterations
```suggestion
  spec.add_runtime_dependency "cocoapods", ">= 1.12.0', '< 2.0"
```

---

@lmcjt37 done